src/main/java/org/candlepin/resource/CdnResource.java

```diff
@@ -44,18 +46,26 @@ public class CdnResource implements CdnApi {
     private CdnCurator curator;
     private CdnManager cdnManager;
     private ModelTranslator translator;
+    private PagingUtilFactory pagingUtilFactory;
 
     @Inject
-    public CdnResource(I18n i18n, CdnCurator curator, CdnManager manager, ModelTranslator translator) {
+    public CdnResource(I18n i18n, CdnCurator curator, CdnManager manager, ModelTranslator translator,
+        PagingUtilFactory pagingUtilFactory) {
         this.i18n = i18n;
         this.curator = curator;
         this.cdnManager = manager;
         this.translator = translator;
+        this.pagingUtilFactory = pagingUtilFactory;
     }
 
     @Override
-    public CandlepinQuery<CdnDTO> getContentDeliveryNetworks() {
-        return this.translator.translateQuery(curator.listAll(), CdnDTO.class);
+    public Stream<CdnDTO> getContentDeliveryNetworks() {
+        Collection<Cdn> cdns;
+        cdns = curator.listAllCdns();
+        Stream<CdnDTO> stream = cdns.stream()
+            .map(this.translator.getStreamMapper(Cdn.class, CdnDTO.class));
+        return this.pagingUtilFactory.forClass(CdnDTO.class)
```

Comment:
Based on [this comment](https://github.com/candlepin/candlepin/pull/4542#discussion_r1479617294) from @nikosmoum, we only need to add paging to endpoints which are known to use paging today. If this isn't one of them, we should pull out the paging util usage.

Reply:
We don't need to have paging on this endpoint, so yes, lets remove the paging util